packages/block-library/src/query/edit/query-inspector-controls.js

```diff
@@ -124,14 +124,14 @@ export default function QueryInspectorControls( { query, setQuery } ) {
 		onChangeDebounced();
 		return onChangeDebounced.cancel;
 	}, [ querySearch, onChangeDebounced ] );
-	const stickyOptions = useMemo( () => [
-		{
-			label: __( 'Include' ),
-			value: '',
-		},
-		{ label: __( 'Exclude' ), value: 'exclude' },
-		{ label: __( 'Only' ), value: 'only' },
-	] );
+	const stickyOptions = useMemo(
+		() => [
+			{ label: __( 'Include' ), value: '' },
+			{ label: __( 'Exclude' ), value: 'exclude' },
+			{ label: __( 'Only' ), value: 'only' },
+		],
+		[]
+	);
```

Comment:
Why is it memoized if it has no dependencies? It can be a constant?

Reply:
Yeah, you're right.

Reply:
The constant does have to be moved out of the component though, otherwise it will be a new array on every render.

Reply:
Okay, I moved it outside. I'm trying to figure out when is best to have it outside and therefore reserve memory, when to useMemo even if no deps to avoid the above memory but keep the same array reference and when it's okay to leave a constant like this inside and rely on React being fast on handling this..  This is a small change though and some performance measurement will be made later afterwards that more code will be added.